autoload/lsp.vim

```diff
@@ -350,7 +350,11 @@ function! s:ensure_init(buf, server_name, cb) abort
     if has_key(l:server_info, 'capabilities')
         let l:capabilities = l:server_info['capabilities']
     else
-        let l:capabilities = {}
+        let l:capabilities = {
+        \   'workspace': {
+        \       'applyEdit ': {'_TYPE': v:msgpack_types.boolean, '_VAL': 1}
```

Comment:
don't think `v:msgpack_types.boolean` works in vim.
Also is this right?

Reply:
Why not do this instead. `applyEdit: v:true`

Reply:
I tested on neovim and it worked there, I was not aware of the `v:true` it is simple and better  :)